core/language.go

```diff
@@ -5,13 +5,25 @@ import (
 	sitter "github.com/smacker/go-tree-sitter"
 )
 
+// ImportContents represents the contents of an import node
+// It represents the ready to use content of import nodes which may
+// exhibit different forms in different languages
+type ImportContents struct {
+	ModuleName  string
+	ModuleItem  string
+	ModuleAlias string
+}
+
 // LanguageResolvers define the minimum contract for a language
 // implementation to resolve language specific concerns
 // such as imports, functions, etc.
 type LanguageResolvers interface {
 	// ResolveImports returns a list of import statements
 	// identified from the parse tree
 	ResolveImports(tree ParseTree) ([]*ast.ImportNode, error)
+
+	// ResolveImportContents returns the contents of an import
+	ResolveImportContents(importNode *ast.ImportNode) (ImportContents, error)
```

Comment:
Lets not introduce this because its not a core language concern. As discussion in https://github.com/safedep/code/pull/21#discussion_r1974620041

Let us leave it on client code to decide how to deal with a language specific import name. For Go, the language specific import names include double quotes.